LEAF_Request_Portal/scripts/automated_email.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+
+$currDir = dirname(__FILE__);
+
+include_once $currDir . '/../db_mysql.php';
+include_once $currDir . '/../db_config.php';
+
+require_once $currDir . '/../Email.php';
+require_once $currDir . '/../VAMC_Directory.php';
+ini_set('display_errors',1);
+error_reporting(E_ALL);
+
+// copied from FormWorkflow.php just to get us moved along.
+$protocol = 'https';
+$siteRoot = "{$protocol}://" . HTTP_HOST . dirname($_SERVER['REQUEST_URI']) . '/';
+
+// this was what the random function I found used.
+$comment = '';
+
+$db_config = new DB_Config();
+$db = new DB($db_config->dbHost, $db_config->dbUser, $db_config->dbPass, $db_config->dbName);
+
+$getWorkflowStepsRes = $db->prepared_query('SELECT workflowID, stepID,stepTitle,stepData
+FROM workflow_steps WHERE stepData LIKE \'%"AutomateEmailGroup":"true"%\';', []);
+

```

Comment:
Aren't we pulling the sql statement out of the prepared_query function into a variable and just using the variable inside the function?

Reply:
Moved this around a bit, the variable was a bit tricky since we are dealing with quotes. This seemed like the only way to format it the way I needed it. This string is moved into a separate variable for use here, variable in the query stayed in that string.